packages/turbo-workspaces/src/managers/pnpm.ts

```diff
@@ -125,16 +136,28 @@ async function create(args: CreateArgs): Promise<void> {
  */
 async function remove(args: RemoveArgs): Promise<void> {
   const { project, logger, options } = args;
+  const hasWorkspaces = project.workspaceData.globs.length > 0;
 
-  logger.mainStep(`Removing pnpm workspaces`);
-  if (project.paths.workspaceConfig) {
+  logger.mainStep(
+    `Removing ${project.packageManager}${hasWorkspaces ? "workspaces" : ""}`
```

Comment:
You've missed this space multiple times. We should reword these messages.

```suggestion
    `Removing ${project.packageManager}${hasWorkspaces ? " workspaces" : ""}`
```

Reply:
Yea, was a copy paste - will centralize this